app/components/steps_component/steps_component.scss

```diff
@@ -35,16 +31,25 @@
     h3 {
       margin-bottom: 0;
 
+      @include govuk-media-query($from: tablet) {
+        margin-left: 35px;
+      }
+
       &::before {
+        background-color: govuk-colour('white');
         border: 2px solid $govuk-border-colour;
         border-radius: 50%;
         content: counter(steps-counter);
         display: inline-block;
-        height: 100%;
+        left: 0;
```

Comment:
if this is for the position: absolute in mediaquery below i think it should go in that mediaquery?